README.md

```diff
@@ -1,10 +1,24 @@
 # Lora Based BraveSensor Firmware
 
+# Project Overview
+
+## Setup
+Follow this [guide](https://docs.rakwireless.com/Product-Categories/WisDuo/RAK3272S-Breakout-Board/Quickstart/#rak3272s-breakout-board-as-a-stand-alone-device-using-rui3) on how to setup. The LoraBraveSensor directory can subsitute the Arduino Serial demo, as a guide on how to compile and upload our custom firmware. After going throught the entire process, I recommend using vscode's Arduino extension for easier file management. 
+
+## Secrets File
+DEVEUI: which can be found by scanning the QR code on the LoRaWan chip. 
+APPEUI: currently we use the same APPEUI as the one for BraveButtons
```

Comment:
Did you want to mention that this is in the example in the living doc?

Reply:
I believe I have the link two lines below.

Reply:
Cool. I see it. Thanks!